shell/platform/darwin/ios/BUILD.gn

```diff
@@ -256,6 +256,7 @@ shared_library("ios_test_flutter") {
     "framework/Source/connection_collection_test.mm",
   ]
   deps = [
+    ":flutter_framework",
```

Comment:
What was this change for? I think the tests currently import individual headers instead of the framework.

Reply:
The target depends on this being built to run, but isn't declaring it as a dep. I found this out by building the target before building this dep.